packages/client/src/pages/CheckerPage.tsx

```diff
@@ -255,67 +264,77 @@ const CheckerPage = ({ checker, matomoTrackEvent, resetChecker, topic }) => {
             <LocationInput
               {...{
                 handleNewAddressSubmit,
-                matomoTrackEvent,
-                topic,
               }}
             />
           )}
           {!isActive(sections.LOCATION_INPUT) && (
-            <LocationSummary
-              showEditLocationModal
-              {...{
-                matomoTrackEvent,
-                resetChecker,
-                setActiveState,
-                topic,
-              }}
-            />
+            <LocationSummary showEditLocationModal />
           )}
         </StepByStepItem>
         <StepByStepItem
           active={isActive(sections.QUESTIONS) && questionIndex === 0}
           checked={isFinished(sections.QUESTIONS)}
           customSize
-          done={answers || isActive(sections.QUESTIONS)}
+          done={!isEmtpyObject(answers) || isActive(sections.QUESTIONS)}
           heading="Vragen"
           largeCircle
           // Overwrite the line between the Items
           style={checkedStyle}
         />
         <Questions
           {...{
-            checker,
             goToQuestion,
             isActive,
             isFinished,
-            matomoTrackEvent,
             setActiveState,
             setFinishedState,
-            topic,
           }}
         />
         <StepByStepItem
           active={isActive(sections.CONCLUSION)}
-          as="div"
           checked={isFinished(sections.QUESTIONS)}
+          // as="div" // XXX
```

Comment:
I know this was needed for something. Why is this commented?

Reply:
Good one, I added XXX so we wouldn't forget. I know you build this `as` prop very carefully, however I couldn't make it work with TypeScript. I'll add it to the list, feel free to refactor 👍

Reply:
Fixed in my #686